src/app.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = (file, copy) => {
+  if (file === copy) {
+    // eslint-disable-next-line no-console
+    console.log('Error: Cant create copy with same name!');
+
+    return;
+  }
+
+  try {
+    fs.copyFileSync(file, copy);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error);
```

Comment:
Consider throwing an error instead of using console.log, e.g., 'throw new Error('Failed to copy file.');'.